functions/clients/rawg.go

```diff
@@ -24,3 +24,19 @@ func NewRawg(rawgToken string) *Rawg {
 func (r *Rawg) GetGame(ctx context.Context, game string) (*rawg.GameDetailed, error) {
 	return r.client.GetGame(ctx, game)
 }
+
+func RawgErrorMsg(err error) string {
+	rawgError, ok := err.(*rawg.RawgError)
+	if ok {
+		switch rawgError.HttpCode {
+		case http.StatusNotFound:
+			return "Bot not connected to servers at the moment"
+		case http.StatusUnauthorized:
+			return "RAWG API not availible at the moment"
+		default:
+			return "Unspecified HTTP error occoured"
+		}
+	} else {
+		return "Unspecified error occoured"
+	}
```

Comment:
Messages don't match the error code. I'm also not sure we should show the error to the user.